### ISSUE TITLE
Exception during deepUpdate using withDefault()

### ISSUE THREAD

**Issue by jhuntoo** [AlexGalays/immupdate#10]
Hi there, thanks for the awesome library. I'm not sure if this is a bug or not however i've observed the following when trying to deep update a map using `withDefault`, which I'd like to use to set default values for an object, just once.

```typescript
interface Panel {
  id?: string;
  isCollapsed?: false;
}
const DefaultPanel: Panel = { isCollapsed: false};
interface PanelCollection {
    panels: {[panelId: string]: Panel}
}

const initial: PanelCollection = {
  panels: {}
};
const updatedPanel = {id: 'xxxx-xxxxx' };
const updated = deepUpdate(initial).at('panels').at(updatedPanel.id).withDefault(DefaultPanel).modify(p => { return {...p, ...updatedPanel} });
```

Here is the stacktrace

> immupdate.js:96 Uncaught TypeError: Cannot read property 'abort' of undefined
    at Updater.cloneForUpdate (immupdate.js:96)
    at doModify (immupdate.js:53)
    at Updater.modify (immupdate.js:60)
    at Object.527 (reducers.ts:48)
    at __webpack_require__ (bootstrap f93ece5…:52)
    at Object.572 (app.module.ts:78)
    at __webpack_require__ (bootstrap f93ece5…:52)
    at Object.571 (app.component.ts:104)
    at __webpack_require__ (bootstrap f93ece5…:52)
    at Object.518 (src async:110)
    at __webpack_require__ (bootstrap f93ece5…:52)
    at Object.1099 (actions.ts:86)
    at __webpack_require__ (bootstrap f93ece5…:52)
    at webpackJsonpCallback (bootstrap f93ece5…:23)
    at main.bundle.js:1

This however does seem to work, though just less efficiently
```typescript
deepUpdate(initial).at('panels').at(updatedPanel.id).modify(p => { return {...DefaultPanel, ...p, ...updatedPanel} });
```

Does this look like an easy fix ?

**Comment by AlexGalays** [comment]
Strange, I would have thought one of the tests would cover this scenario, but should be an easy fix ! Thanks for the report.

**Comment by AlexGalays** [comment]
Fixed in 1.1.1  :)

**Comment by jhuntoo** [comment]
@AlexGalays Thanks for super quick turn around!
